src/homework-05/homework-05.ts

```diff
@@ -0,0 +1,41 @@
+// 1. Вам потрібно створити тип DeepReadonly який буде робити доступними тільки для читання навіть властивості вкладених обʼєктів.
+
+type ReadonlyMember<T> = {
+  readonly [P in keyof T]: T[P];
+};
+
+type DeepReadonly<K extends keyof any, U> = {
```

Comment:
While it's not a mistake, I recognize that you can successfully create these mapped types all at once.